grammars/supercollider.cson

```diff
@@ -69,7 +69,7 @@
     'match': '^\\s*(\\**[a-z]{1}[a-zA-Z0-9_]+)\\s*\\{'
   }
   {
-    'match': '\\~[a-zA-Z0-9]+'
+    'match': '\\~[a-zA-Z0-9_]+'
```

Comment:
thanks! this is close, but not correct. `~_` isn't a valid environment variable name, nor is `~_a`. i think it should be `\\~[a-zA-Z0-9][a-zA-Z0-9_]*` then, right? (not 100% sure this notation works like the regex dialect i'm familiar with). the Supercollider IDE highlights `~_a`, i think that ought to be emended